tensorbay/client/version.py

```diff
@@ -14,6 +14,17 @@
 from tensorbay.client.struct import Branch, Commit, Draft, Tag
 from tensorbay.exception import ResourceNotExistError, StatusError
 
+_KEY_TO_TITLES = {
+    "segment": "Segment",
+    "withLabel": "是否包含标注",
+    "frame": "Frame",
+    "sensor": "Sensor",
+    "size": "数据大小",
+    "category": "Category",
+    "attribute": "Attribute",
+    "keyword": "数据名称",
```

Comment:
Chinese character is not recommended in SDK

Reply:
Yes, this title is different in cn and com. So SDK do not need to add this param.

Reply:
What I mean is Chinese character is not recommended in this repo.
Why we need that?

Reply:
The title of filter will be used for display in the frontend, and it is different in cn and com. In my opinion, OpenAPI should add a default value for the `title`. And SDK do not need add this param, because it's useless to the user.